app/src/main/java/ca/bc/gov/bchealth/ui/resources/ResourcesFragment.kt

```diff
@@ -18,27 +19,55 @@ class ResourcesFragment : Fragment(R.layout.fragment_resources) {
 
     private val binding by viewBindings(FragmentResourcesBinding::bind)
     private val analyticsFeatureViewModel: AnalyticsFeatureViewModel by viewModels()
+    private lateinit var resourcesAdapter: ResourcesAdapter
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
         setupToolBar()
+        setupRecyclerView()
+    }
 
-        binding.customView1.setOnClickListener {
-            val url = getString(R.string.url_how_to_get_covid_vaccinated)
-            requireActivity().redirect(url)
-            seedAnalyticsData(url = url)
-        }
-        binding.customView2.setOnClickListener {
-            val url = getString(R.string.url_get_tested_for_covid)
-            requireActivity().redirect(url)
-            seedAnalyticsData(url = url)
-        }
-        binding.customView3.setOnClickListener {
-            val url = getString(R.string.url_covid_symptom_checker)
+    private fun prepareResourcesList(): MutableList<Resources> {
```

Comment:
This resource list should be in viewModel

Reply:
This has been taken care.